AUTHORS.md

```diff
@@ -0,0 +1,17 @@
+SvgLib was designed and developed by Fabien Ménager.
```

Comment:
maybe .gitattributes this file

Reply:
I wonder if we should keep the "about" files for reference. SECURITY.md is excluded, but now I'm not sure why. Any good guidance on if any of those files should remain or be excluded on export?

Reply:
Well, I think for this you need to think that each file that is not needed by code will be installed in each software, server, dev pc that uses this software one way or another. Many softwares do not cleanup vendor folders and distribute such files. For example Lime Survey written in PHP. It's a snowball effect that takes disk space 😸

Reply:
I'm going to leave this file for the time being to maintain consistency with the other Dompdf projects. I'll need to think through the optimal export structure.